dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.health-4.0.feature

```diff
@@ -0,0 +1,9 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
The previous private feature files were named with `com.ibm.websphere.appserver.*`.  Are we changing it to `io.openliberty.*`?

Reply:
Yes, for the MP 5.0 features, we are changing the private feature files to `io.openliberty.*`.